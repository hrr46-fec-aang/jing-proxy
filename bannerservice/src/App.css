body{
  font-family: sans-serif;
}
img {
  height: 30px;
  width: 30px;
}

.banner-search {
  flex: 0 0 auto;
  position: relative;
  display: flex;
  justify-content: flex-end;
}

.banner-search__input {
  font-size: 14px;
  width: 0;
  max-width: calc(100vw - 8.5em);
  padding: 0;
  border: none;
  background: transparent;
  text-indent: 0.5em;
  margin-right: 0;
  color: white;
  text-shadow: 0 0 0 #666;
  transition: all 250ms;
}

.banner-search__input::-webkit-input-placeholder{
  color: #666;
  text-shadow: none;
  -webkit-text-fill-color: initial;
}

.banner-search__icon {
  font-size: 1em;
  width: 1em;
  height: 1em;
  padding: 0;
  border: none;
  cursor: pointer;
  fill: lightgrey;
  background: transparent;
  transition: color 250ms;
}

.banner-search__input:focus,
.banner-search__icon:focus {
  outline: none;
}

.banner-search__clear {
  display: none;
  margin-right: 0.25em;
  margin-bottom: -1px;
}

.banner-search--active .banner-search__input {
  margin-right: 0.5em;
  width: 400px;
}

.banner-search--active .banner-search__clear {
  display: block;
}

.banner-search__results {
  position: absolute;
  top: 100%;
  right: 0;
  width: 100%;
  margin-top: 5px;
}

.banner-sub {
  display: block;
  background: white;
}

.banner-sub__inner {
  min-width: 320px;
  max-width: 960px;
  margin: 0 auto;
  padding: 0 1.5em;
  text-align: right;
}

.banner-sub__link {
  display: inline-block;
  font-size: 0.8em;
  padding: 0.4em 0;
  color: black;
  text-transform: uppercase;
  text-decoration: none;
}

.banner-sub__link:not(:last-child) {
  margin-right: 1.5em;
}

.banner-sub__link:first-of-type {
  margin-left: 0;
}

@media (max-width: 720px) {
  .banner-sub {
      display: none;
  }
}
.banner {
  position: relative;
  background: white;
  display: inline;
}

.banner__inner {
  display: flex;
  width: 100%;
  min-width: 320px;
  max-width: 960px;
  min-height: 70px;
  margin: 0 auto;
  padding: 0 1.5em;
  align-items: center;
  background-image: url('https://s3-us-east-2.amazonaws.com/rvshare-wordpress/wp-content/uploads/2019/05/15171042/hipcamp-logo-1024x1024.png');
  background-size: 30px;
  background-repeat: no-repeat;
  background-position: 0% 40%;


}

.banner__mobile {
  display: none;
  fill: white;
  padding: 0;
  margin: 0;
  line-height: 0;
  outline: none;
  border: none;
  cursor: pointer;
  background: transparent;
  transition: color 250ms;
}

.banner__mobile svg {
  width: 20px;
  height: 20px;
}

.banner__logo {
  margin: auto auto auto 12px;
  text-decoration: none;
  text-transform: uppercase;
  font-weight: bolder;
  font-size: x-large;
  letter-spacing: 0.5px;
  color: black;
  transition: all 250ms;
}
.banner__logo:hover {
  color: #40d9ac;
}

.banner__items {
  flex: 1 1 auto;
  display: flex;
  align-items: center;
  justify-content: flex-end;
}

.banner__item {
  display: inline-block;
  font-size: 0.8em;
  padding: 1.5em;
  cursor: pointer;
  color: black;
  text-transform: uppercase;
  text-decoration: none;
  transition: all 250ms;
  font-weight: bold;
}

.banner__item:hover {
color:  #40d9ac;
}

.banner__item--active {
  background: white;
  color: black;
}

.banner__item--offset {
  margin-right: -1.5em;
}

.banner--search .banner__item {
  pointer-events: none;
  overflow: hidden;
  white-space: nowrap;
  padding: 1.5em 0;
  margin-right: -35px;
  opacity: 0;
}

.banner--overlay {
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  z-index: 50;
  background: transparent;
}

@media (max-width: 720px) {
  .banner__mobile { display: block; }
  .banner__items { display: none; }
  .banner--search .banner__logo {
      pointer-events: none;
      overflow: hidden;
      white-space: nowrap;
      text-overflow: ellipsis;
      margin-right: -35px;
      letter-spacing: -1em;
      opacity: 0;
  }
}

