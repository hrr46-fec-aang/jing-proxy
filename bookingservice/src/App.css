.container {
  position: relative;
}

.booking {
  text-align: center;
  font-family: 'Open Sans', sans-serif;
  font-weight: 600;
  background-color: rgb(255, 255, 255);
  max-width: 20em;
  min-width: 20em;
  margin: 50px;
  border-style: solid;
  border-color: #ebebeb;
  border-width: 0.01em;
  position: absolute;
  top: 50%;
  right: 16px;
  z-index: 2;
}
h2 {
  margin: 7px 5px 0px 15px;
  text-align: left;
}

input[type='number']:focus {
  outline: none;
  border-style: solid !important;
  border-color: #40d9ac;
  border-radius: 5px;
}
input[type='text']:focus {
  outline: none;
  border-style: solid !important;
  border-color: #40d9ac;
  border-radius: 5px;
}
.grid-container {
  display: grid;
  grid-template-columns: 35% 35% 30%;
  grid-template-rows: 80px;
  background-color: white;

  border-top: solid;
  border-bottom: solid;
  border-color: #ebebeb;
  border-width: 0.01em;
}

.grid-container > div {
  background-color: rgba(255, 255, 255, 0.8);
  text-align: center;
  padding: 20px 0px 0px 0px;
  font-size: 15px;
}
input:focus {
  background-color: lightgray;
}

input,
#checkinCalendar {
  width: 5em;
  margin: 0px 11px 0px 0px;
  text-align: left;
  border-style: none;
}

input,
#checkoutCalendar {
  width: 5em;
  margin: 0px 11px 0px 0px;
  text-align: left;
  border-style: none;
}
input,
#guestCount {
  width: 5em;
  text-align: center;
  border-style: none;
}

.book {
  border-top: solid;
  border-color: #ebebeb;
  border-width: 0.01em;
  padding: 10px;
}

#pernight {
  text-align: left;
  font-size: small;
  font-weight: 300;
  color: #757575;
  padding: 0px 0px 10px 0px;
  margin: 0px 0px 0px 15px;
}

.checkintxt {
  text-align: left !important;
  margin: 0px 0px 0px 15px !important;
}
.checkouttxt {
  text-align: left !important;
  margin: 0px 0px 0px 14px !important;
}
.checkout {
  border-left: solid;
  border-right: solid;
  border-color: #ebebeb;
  border-width: 0.01em;
}
.loader {
  margin: 11px 0px 11px 0px;
}

.subtotal {
  padding: 11px 0px 11px 0px;
}

.react-datepicker__day--selected,
.react-datepicker__day--in-range {
  background-color: #40d9ac;
}
.react-datepicker__day--disabled {
  background-color: white;
}
.react-datepicker__day--keyboard-selected {
  background-color: white;
  color: black;
}
.react-datepicker__day--in-selecting-range:hover,
.react-datepicker__day--keyboard-selected:hover,
.react-datepicker__day--in-selecting-range {
  background-color: #40d9ac;
}

input[type='number']::-webkit-inner-spin-button {
  opacity: 1;
}
