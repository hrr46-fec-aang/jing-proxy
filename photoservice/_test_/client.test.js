/**
 * @jest-environment jsdom
 */
import { shallow, mount, render } from 'enzyme';
import React from 'react';
import sinon from 'sinon';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

import App from '../client/component/app/app.jsx';

describe('App Component', () => {
  let component;

  beforeAll(() => {
    component = mount(<App />);
  });

  afterAll(() => {
    component.unmount();
  });

  it('should render correctly with no props', () => {
    expect(component).toMatchSnapshot();
  });
});
