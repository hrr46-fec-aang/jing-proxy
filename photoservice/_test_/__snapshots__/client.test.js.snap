// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should render correctly with no props 1`] = `ReactWrapper {}`;
